### ISSUE TITLE
508-Defect-1 [COGNITIVE] Direct Deposit Information box functionality

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#28438]
[508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Noah

## User Story or Problem Statement

The functionality of the "Direct Deposit Information" box is not accessible and there are a number of UI issues

### When the user has account information and checks "I don't want to use direct deposit" there is no way to restore the account details

If the user were to accidentally select the "I don't want to use direct deposit" their account information in the form is permanently erased and cannot be undone. Also, when the user goes to uncheck it again, the Direct Deposit Information box expands and all fields are in an error state. This causes the screen reader to announce all the errors at once without giving the user the ability to fill them in first.

Developer should add a way to restore account details if accidentally erased as well as prevent the form from appearing in an error state by default

![508-3](https://user-images.githubusercontent.com/984252/123379047-6260c900-d542-11eb-821e-c5b048253a5e.PNG)

## Acceptance Criteria

- [ ] Allow users to restore account information
- [ ] Do not load form in an error state by default

**Comment by zurbergram** [comment]
@noahgelman I believe that the removal of direct deposit info is an intentional design choice.  

The changes in screenreader behavior will involve digging into platform code as [platform/forms-system/src/js/definitions/directDeposit](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/definitions/directDeposit.jsx) is form config based and not a custom component.

Any changes to direct deposit will affect any form using [platform/forms-system/src/js/definitions/directDeposit](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/definitions/directDeposit.jsx)

Recommending passing this on to platform to review

**Comment by af-manish** [comment]
@noahgelman  Where do we stand on this with platform?
